lib/fhir_models/fhir_ext/structure_definition.rb

```diff
@@ -426,7 +426,7 @@ def valid_json?(json)
           else
             element.max.to_i
           end
-        if (nodes.size < min) && (nodes.size > max)
+        if (nodes.size < min) || (nodes.size > max)
```

Comment:
`min` and `max` define a Range.  Perhaps it would be better to use native ruby concepts.
```
(min..max).cover?(nodes.size)

or 

(min..max).include?(nodes.size)
```